dapps/ImportTorusMM.md

```diff
@@ -7,7 +7,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import styles from '../src/css/custom.css'
 
 
-While the Torus wallet is a great option for newcomers to crypto, it can be more difficult to use when interacting with decentralized applications (dApps). Giveth users have the opportunity to earn [GIV](https://docs.giveth.io/giveconomy/) by interacting with the [GIVeconomy](https://giveth.io/). Torus users can claim [GIVbacks](https://giveth.io/givbacks), stake in the [GIVfarm](https://giveth.io/givfarm), claim their [GIVstream](https://giveth.io/givstream) and [claim their GIVdrop](https://giveth.io/claim) ([if eligible](https://docs.giveth.io/giveconomy/givdrop)), but MetaMask users benefit from a more streamlined user experience.  In addition, at present the [GIVgarden](https://gardens.1hive.org/#/xdai/garden/0xb25f0ee2d26461e2b5b3d3ddafe197a0da677b98) does not include Torus wallet integration.
+While the Torus wallet is a great option for newcomers to crypto, it can be more difficult to use when interacting with decentralized applications (dApps). Giveth users have the opportunity to earn [GIV](https://docs.giveth.io/giveconomy/) by interacting with the [GIVeconomy](https://giveth.io/). Torus users can claim [GIVbacks](https://giveth.io/givbacks), stake in the [GIVfarm](https://giveth.io/givfarm), claim their [GIVstream](https://giveth.io/givstream) and [claim their GIVdrop](https://giveth.io/claim) ([if eligible](https://docs.giveth.io/giveconomy/givdrop)), but MetaMask users benefit from a more streamlined user experience.
```

Comment:
they can't claim their GIVdrop either, we could take that out. idk how to just edit that here @divine-comedian